app/src/main/java/com/mediaan/mymediaan/view/ftu/CreateProfileScreen.kt

```diff
@@ -0,0 +1,305 @@
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.DrawerValue
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ExposedDropdownMenuBox
+import androidx.compose.material3.ExposedDropdownMenuDefaults
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.rememberDrawerState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import androidx.navigation.compose.rememberNavController
+import com.mediaan.mymediaan.R
+import com.mediaan.mymediaan.model.TwoTruthsOneLieEntity
+import com.mediaan.mymediaan.repository.ProfileRepository
+import com.mediaan.mymediaan.ui.theme.MediaanPrimary
+import com.mediaan.mymediaan.ui.theme.MyMediaanTheme
+import com.mediaan.mymediaan.ui.theme.Typography
+import com.mediaan.mymediaan.view.MyMediaanAppBar
+import com.mediaan.mymediaan.viewModel.MainNavigationViewModel
+import com.mediaan.mymediaan.viewModel.MyMediaanScreen
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun CreateProfileScreen(
+    navController: NavController,
+    viewModel: MainNavigationViewModel,
+    profileRepository: ProfileRepository,
+) {
+    Scaffold(
+        topBar = {
+            MyMediaanAppBar(
+                drawerState = rememberDrawerState(initialValue = DrawerValue.Closed),
+                title = stringResource(MyMediaanScreen.Ftu.title),
+                navIcon = Icons.Filled.ArrowBack,
+                iconDescr = "Back",
+                onNavIconClick = { navController.popBackStack() }
+            )
+        },
+    ) { innerPadding ->
+        val scrollState = rememberScrollState()
+        var firstName by remember { mutableStateOf("") }
+        var nickName by remember { mutableStateOf("") }
+        var lastName by remember { mutableStateOf("") }
+        val minCharName = 3
+        val maxCharName = 50
+        var age by remember { mutableStateOf("") }
+        val minAge = 18
```

Comment:
Maybe better to put it at 16 or something, I'd assume at some point an intern is underage - but I could be wrong.